dali/sasha/saserver.cpp

```diff
@@ -322,6 +323,9 @@ int main(int argc, const char* argv[])
         lf->setMaxDetail(TopDetail);
         lf->beginLogging();
     }
+#else
+        PROGLOG("Saserver NOT logging to local file!");
```

Comment:
should probably call the same common helper routine e.g. startComponentLogging()